.gitignore

```diff
@@ -0,0 +1,6 @@
+*.iml
```

Comment:
to be fair, rather add these to your local .git/info/exclude file to save this getting littered with everyones ide project files